Makefile

```diff
@@ -179,10 +179,12 @@ tmp/weave-scope.tgz: $(shell find client/app -type f) $(SCOPE_UI_BUILD_UPTODATE)
 else
 
 client/build/index.html:
-	cd client && npm run build
+	test "true" = "$(SCOPE_SKIP_UI_ASSETS)" && { mkdir -p client/build; } || \
```

Comment:
I've never seen braces used like this. Is it for subprocess execution? What's the difference between `{ mkdir -p client/build; }` and `( mkdir -p client/build; )`?